conf/messages

```diff
@@ -2782,3 +2782,59 @@ login-event.skip-for-now = Skip for now and continue to the service
+accessibility.statement.paragraph.six = This service is run by HM Revenue and Customs (HMRC). We want as many people as possible to be able to use this service. This means you should be able to:
+accessibility.statement.paragraph.seven = We have also made the text in the service as simple as possible to understand.
+accessibility.statement.paragraph.eight = has advice on making your device easier to use if you have a disability.
+accessibility.statement.paragraph.eight.link = AbilityLink
+accessibility.statement.paragraph.nine = This service is fully compliant with the
+accessibility.statement.paragraph.nine.link = Web Content Accessibility Guidelines version 2.1 AA standard
+accessibility.statement.paragraph.ten = There are no known accessibility issues within this service.
+accessibility.statement.paragraph.eleven = If you have difficulty using this service, contact us by:
+accessibility.statement.paragraph.twelve = We are always looking to improve the accessibility of this service. If you find any problems that are not listed on this page or think we are not meeting accessibility requirements,
+accessibility.statement.paragraph.twelve.link = report the accessibility problem
+accessibility.statement.paragraph.thirteen1 = The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you are not happy with how we respond to your complaint,
+accessibility.statement.paragraph.thirteen2 = or the
+accessibility.statement.paragraph.thirteen3 = if you live in Northern Ireland
+accessibility.statement.paragraph.thirteen.link1 = contact the Equality Advisory and Support Service (EASS)
+accessibility.statement.paragraph.thirteen.link2 = Equality Commission for Northern Ireland (ECNI)
+accessibility.statement.paragraph.fourteen = We provide a text relay service if you are deaf, hearing impaired or have a speech impediment.
+accessibility.statement.paragraph.fifteen = We can provide a British Sign Language (BSL) interpreter, or you can arrange a visit from an HMRC advisor to help you complete the service.
+accessibility.statement.paragraph.sixteen = Find out how to
+accessibility.statement.paragraph.sixteen.link = contact us
+accessibility.statement.paragraph.seventeen = HMRC is committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+accessibility.statement.paragraph.eighteen = This service is fully compliant with the
+accessibility.statement.paragraph.eighteen.link = Web Content Accessibility Guidelines version 2.1 AA standard
+accessibility.statement.paragraph.nineteen = The service was last tested on Monday {0} and was checked for compliance with WCAG 2.1 AA.
```

Comment:
... last tested on Monday... what if the date will be Tuesday?

Reply:
Changed